src/Proj4.jl

```diff
@@ -125,8 +130,9 @@ function transform!(src::Projection, dest::Projection, position::Array{Float64,2
     P = pointer(position)
     x = P
     y = P + sizeof(Cdouble)*npoints
-    z = ncomps < 3 ? C_NULL : P + 2*sizeof(Cdouble)*npoints
-    err = _transform!(src, dest, npoints, 1, x, y, z)
+    z = P + 2*sizeof(Cdouble)*npoints
+    err = (ncomps < 3) ? _transform!(src, dest, npoints, Int32(1), x, y) :
+          _transform!(src, dest, npoints, Int32(1), x, y, z)
```

Comment:
I was not aware of it, thanks! Seems to work only for julia v0.4 though, But you're right, I've removed the redundant method